### PR TITLE
NF-related additions

### PR DESCRIPTION
**Pull request by sgosline**
A few updates to account for NF data

### CODE REVIEW

**Comment by sgosline** [comment]
updated to move assayTarget

**Comment by karawoo** [comment]
@sgosline is Dorsal Root Ganglion really a cell type, or a cluster of cells? Most of the other entries in `cellType` look like types of individual cells, but I'm not really sure...

**Comment by sgosline** [comment]
That's a good point, the full cells are "DRG/nerve root neurosphere cells"

**Comment by sgosline** [comment]
So maybe DRG is the tissue...

**Comment by karawoo** [comment]
That makes more sense to me but it might be good to get confirmation from someone else. Everything else in this PR looks good to me so if you want to pull out that one change for now I will approve it ✅

**Comment by sgosline** [comment]
Hmm, I'll tag @allaway. Robert, the description of the cells is here: https://www.synapse.org/#!Synapse:syn8016635/wiki/411638

**Comment by allaway** [comment]
DRG is structured and comprised of  multiple cell types, so I think `tissue` is appropriate for it (although NCIT considers it a `part` of the PNS). 

This paper from Lu's lab describes the model in more detail. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4254535/

As I understand it, DRG/nerve root neurosphere cells are still probably comprised of multiple cell types/subtypes. Since it's only the embryonic GAP43+PLP+ Schwann cell precursors that are being isolated and expanded, perhaps `embryonic GAP43+PLP+ Schwann cell precursor` is the most descriptive cell type. `Schwann cell precursor` could also be sufficient - that's what this group calls GAP43+PLP+ SC precursor cells: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5469943/.

Cell Ontology has that as a listed term: http://purl.obolibrary.org/obo/CL_0002375

File: synapseAnnotations/data/experimentalData.json [hunk 1]
```diff
@@ -937,6 +1005,11 @@
     "columnType": "STRING",
     "maximumSize": 250,
     "enumValues": [
+      {
+        "value": "Dorsal Root Ganglion",
```
Comment (allaway) [inline synapseAnnotations/data/experimentalData.json]: Move to "tissue"

File: synapseAnnotations/data/experimentalData.json [hunk 2]
```diff
@@ -937,6 +1005,11 @@
     "columnType": "STRING",
```
Comment (allaway) [inline synapseAnnotations/data/experimentalData.json]: add `Schwann cell precursor`

http://purl.obolibrary.org/obo/CL_0002375

**Pull request merged**